### ISSUE TITLE
404 and 500 error pages

### ISSUE THREAD

**Issue by ltartari** [okfn-brasil/mosaico#7]

**Comment by vitorbaptista** [comment]
@ltartari Is this done?

**Comment by ltartari** [comment]
No. I just added the html files, but couldn't point the routes. Can you take a look for me on that, @vitorbaptista ?

After that the only thing missing will be the design and content for the pages.
